independent-projects/qute/core/src/main/java/io/quarkus/qute/TemplateInstance.java

```diff
@@ -37,6 +37,11 @@ public interface TemplateInstance {
      */
     String LOCALE = "locale";
 
+    /**
+     * Attribute key - the initial capacity of the StringBuilder used to render the template.
+     */
+    String CAPACITY = "capacity";
```

Comment:
Beware - it is not `static final` - which means the other "constants" in `TemplateInstance` need to be fixed (same PR other commit if you wish!)

Reply:
Maybe not - let me double check - my brain maybe is cheating me

Reply:
And indeed morning brain cheated me :"( removing the comments